lib/thumbs/pull_request_worker.rb

```diff
@@ -702,7 +702,10 @@ def create_build_status_comment
 
 <%= status[:command] %>
 
-<%= status[:output] %>
+<% output=status[:output] %>
+<% allowed_length=10000 %>
+<%= output.slice!(output.length-allowed_length, output.length) %>
```

Comment:
Oh yeh, thats why added the thumbot wait lock (not implemented yet :)  I'm tweaking it now to show "..snipped N lines.."